README.md

```diff
@@ -2,15 +2,211 @@
+
+### 1.b create a simlink instead of copy
+```shell
+ln -s ${HOME}/.kube/config ./bootstrap/assets/kubeconfig 
+```
+
+### 2.a Create a Development Vault Server
+Install vault binary from Homebrew
+```shell
+brew install vault
+```
+Reference:
+* https://www.vaultproject.io/
+
+Outputs:
+```console
+To restart vault after an upgrade:
+  brew services restart vault
+  Or, if you don't want/need a background service you can just run:
+  /opt/homebrew/opt/vault/bin/vault server -dev
+  ==> Summary
+  🍺  /opt/homebrew/Cellar/vault/1.12.2: 8 files, 202.4MB
+  ==> Running `brew cleanup vault`...
+```
+
+### 2.b Start Vault Server in Dev mode
+```shell
+vault server -dev -dev-root-token-id="education"
+```
+
+### 3. Export your new Vault address to the shell
+```shell
+export VAULT_ADDR='http://127.0.0.1:8200'
+```
+check the status, before this the `VAULT_ADDR` must be exported
+```shell
+vault status
+```
+### 4. Initialize your infrastructure
+Homebrew install terraform
+```shell
+brew install terraform
+```
+Reference:
+* https://www.terraform.io/
+
+Output:
+```console
+==> Running `brew cleanup terraform`...
+Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
+Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
+```
+
+```shell
+pushd ./bootstrap;
+terraform init -upgrade;
+popd
+```
+
+```shell
+pushd ./bootstrap;
+terraform apply;
+```
+* Enter the token: "mytoken"
+* Enter the url: http://127.0.0.1:8200
+* Enter Terraform perform actions confirmation: yes
+
+Clear the directory stack
+```shell
+popd
+```
+### 5. Check your infrastructure (`minikube`,`k9s`, `kubectl`, Lens...)
+With ArgoCD
+or Openlens
+or kubectl
+
+### 5.a show namespaces
+```shell
+kubectl get namespaces
+```
+Output:
+````console
+NAME              STATUS   AGE
+apps-system       Active   14m
+argocd            Active   14m
+default           Active   99d
+kube-node-lease   Active   99d
+kube-public       Active   99d
+kube-system       Active   99d
+secrets-system    Active   13m
+````
+
+Show Secrets
+```shell
+kubectl -n secrets-system get all
+```
+Output:
+```
+NAME                                                          DESIRED   CURRENT   READY   AGE
+replicaset.apps/external-secrets-78986ddfdf                   1         1         1       14m
+replicaset.apps/external-secrets-cert-controller-79688fbc89   1         1         1       14m
+replicaset.apps/external-secrets-webhook-887d856fb            1         1         1       14m
+```
+### 6. Issue
+An Output of issue can be found at `Output.md`
+[Execution log](Output.md)
+
+There is also Warning from "externalsecret-validate" with message "ca cert not yet ready" from object `
+validating-webhook-configuration`
+
+### 7. Stop and clean up all
+#### 7.a kill vault server
+```shell
+pkill vault;
+ps aux | grep vault | grep -v grep
+```
+
+#### 7.b clean up k8s
+##### 7.b.1. remove the finalizer from "argocd"
+```shell
+kubectl get namespace argocd -o json >tmp.json;
+```
```

Comment:
Everything created in the cluster has been created with Terraform, ergo it should be destroyed with it.

Use `terraform destroy` to clean the cluster (and the Vault KV-Engines, even though they won't be available at this step if you already killed the development server).